lms/djangoapps/learner_home/views.py

```diff
@@ -571,8 +579,17 @@ def get(self, request):
             },
         )
 
-        if is_control or not course_keys:
-            return Response(status=400)
+        if is_control:
+            return Response(
+                {
+                    "courses": settings.GENERAL_RECOMMENDATIONS,
```

Comment:
Note that this will probably also require we revert [this PR which removed that setting](https://github.com/edx/edx-internal/pull/7179).